macao-sdk-mirror/src/commonMain/kotlin/com/macaosoftware/plugin/AuthPlugin.kt

```diff
@@ -11,13 +11,26 @@ interface AuthPlugin : MacaoPlugin {
     suspend fun login(loginRequest: LoginRequest): MacaoResult<MacaoUser>
     suspend fun loginEmailAndLink(loginRequest: LoginRequestForEmailWithLink)
     suspend fun sendEmailLink(loginRequest: LoginRequestForLink)
+    suspend fun checkCurrentUser(): MacaoResult<MacaoUser>
+    suspend fun getUserProfile(): MacaoResult<MacaoUser>
+    suspend fun getProviderData(): MacaoResult<ProviderData>
+    suspend fun updateProfile(displayName: String, photoUrl: String): MacaoResult<MacaoUser>
+    suspend fun updateFullProfile(displayName: String?, country: String?, photoUrl: String?, phoneNo: String?, facebookLink: String?, linkedIn: String?, github: String?): MacaoResult<UserData>
```

Comment:
I don't want to do any social media login. It takes a lot of times and users barely use that. You can remove anything that is social media signin related

Reply:
I didn't add social media logins, just taking data of the user inside the database